config/config.go

```diff
@@ -836,12 +836,11 @@ func (c *ScrapeConfig) Validate(globalConfig GlobalConfig) error {
 		}
 	}
 
+	if model.NameValidationScheme != model.UTF8Validation {
```

Comment:
Why not error (for downstream config users)?

Reply:
because this is really a programming error, not a user error - but yes it seems fine to just leave it as a regular error